docs/xcompilationthreads.md

```diff
@@ -31,9 +31,14 @@ Use this option to specify the number of compilation threads that are used by th
 
         -XcompilationThreads<n>
 
-: where `<n>` is the number of threads, in the range 1-7 inclusive. Any number outside this range is ignored. 
+: Where <n\> is the number of threads. The minimum value is 1. The default and maximum values depend on whether the VM is running with the JITServer feature enabled and if so, whether the VM is running as a JITServer server, as shown in the following table:
 
-: Setting the compilation threads to zero does not prevent the JIT from working. Instead, if you do not want the JIT to work, use the [`-Xint`](xint.md) option.
+| JIT compiler scenario| Default value | Maximum value |
+|----------------------|---------------|:-------------:|
+| JIT and JITServer client| Determined by internal JIT heuristics that consider the number of CPUs that the VM is allowed to use                         | 15            |
+| JITServer server        | 63         | 999           |
+
+: Any number outside this range is ignored. Setting the compilation threads to zero does not disable the JIT compiler. If you want to disable the JIT, use the [`-Xint`](xint.md) option.
 
```

Comment:
The changes look good to me but we could still use a link to the JITServer topic. You could add a "See also" section, which is common to many command line option topics, eg https://www.eclipse.org/openj9/docs/xalwaysclassgc/.